src/plat/bcm2711/overlay-rpi4.dts

```diff
@@ -21,4 +22,25 @@
 		/* This is configurable in the Pi's config.txt, but we use 128MiB of RAM by default. */
 		reg = <0x00 0x00000000 0x08000000>;
 	};
+
+	reserved-memory {
```

Comment:
I think hardware_gen.py should eventually be updated to properly handle `/memreserve/` statements.

Reply:
I made an issue to track this: https://github.com/seL4/seL4/issues/899